### PR TITLE
Fix formatting and with_hasher example

### PR DESCRIPTION
**Pull request by jeromefroe**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -78,18 +78,15 @@ use core::mem;
...
+#[cfg(feature = "hashbrown")]
+use hashbrown::HashMap;
 
 #[cfg(test)]
 #[macro_use]
 extern crate std;
 
-#[cfg(feature = "nightly")]
-extern crate alloc;
-
```
Comment (kornholi) [inline src/lib.rs]: Why was this removed? `lru` no longer builds with the `nightly` feature:

```rs
error[E0433]: failed to resolve: unresolved import
  --> external/third_party__lru__0_4_2/src/lib.rs:71:5
   |
71 | use alloc::borrow::Borrow;
   |     ^^^^^
   |     |
   |     unresolved import
   |     help: a similar path exists: `core::alloc`

error[E0433]: failed to resolve: unresolved import
  --> external/third_party__lru__0_4_2/src/lib.rs:72:5
   |
72 | use alloc::boxed::Box;
   |     ^^^^^
   |     |
   |     unresolved import
   |     help: a similar path exists: `core::alloc`
```

Comment (jeromefroe) [reply src/lib.rs]: Sorry, this should have never been removed, I'll put a fix and yank the old version of the crate with this version.
